### ISSUE TITLE
Ruby reaper incorrectly checks active jobs — removes every active lock as result

### ISSUE THREAD

**Issue by markiz** [mhenrixon/sidekiq-unique-jobs#517]
**Describe the bug**
https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/lib/sidekiq_unique_jobs/orphans/ruby_reaper.rb#L120-L132

This method checks sidekiq process list, and not active jobs. 

Process hash looks like this in our setup:

```
{"hostname"=>"dss-queue-single2", "started_at"=>1591895424.547636, "pid"=>13767, "tag"=>"sg_dss_portal", "concurrency"=>5, "queues"=>["heavy"], "labels"=>[], "identity"=>"dss-queue-single2:13767:db7bbfe1a54e"}
```

In sidekiq dashboard it looks like this:
![image](https://user-images.githubusercontent.com/53611/84422987-81860480-ac26-11ea-89ee-5c9dcf13855b.png)
Note: one the `db7bbfe1a54e` is not the job id, but rather process id.

This results in reaper removing any locks for the jobs being actively executed, so you might see however many concurrent heavy jobs at the same time.

**Comment by markiz** [comment]
https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/api.rb#L925 I think what should have been used is `workers` instead of `info`, and iterate over all jobs. Or maybe use the Sidekiq API itself.

**Comment by mhenrixon** [comment]
Released as [v7.0.0.beta22](https://github.com/mhenrixon/sidekiq-unique-jobs/releases/tag/v7.0.0.beta22)

**Comment by markiz** [comment]
@mhenrixon thank you 🙏

**Comment by mhenrixon** [comment]
@markiz Thank you for the wonderfully detailed bug reports. Made it a breeze to fix

**Comment by godric** [comment]
Hi @mhenrixon, not sure if I should have opened a separate issue..

This bug is mentioned as "fixed" in the [changelog for v6.0.23](https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/CHANGELOG.md#v6023-2020-06-23), but there isn't anything about it in [the corresponding diff](https://github.com/mhenrixon/sidekiq-unique-jobs/compare/v6.0.22...v6.0.23)

Am I correctly assuming that the Changelog is wrong and this bug still present on the 6.x branch? 
If so, is there any plans to backport the fix in 6.x ? 🙏

**Comment by mhenrixon** [comment]
Hi @godric,

Unfortunately, there will be no reaping of old jobs in v6. I am having some trouble with the changelog generator.
